openmeetings-web/src/main/webapp/css/raw-room.css

```diff
@@ -671,7 +671,10 @@ html[dir="rtl"] .room-block .sb-wb .sidebar {
 	right: 40px;
 	bottom: 40px;
 	padding: 5px;
-	background-color: var(--bs-info);
+	border: 1px solid;
+	border-radius: 0.2rem;
+	background-color: white;
```

Comment:
Would this hardcode looks good in case for ex. this https://bootswatch.com/darkly/ theme is selected?
Should we use some `--bs-` color variable here?

Reply:
True I'll change it to --bs-light